packages/salesforcedx-vscode-lwc/src/index.ts

```diff
@@ -79,6 +79,21 @@ export async function activate(context: vscode.ExtensionContext) {
   context.subscriptions.push(commands);
 }
 
+// See https://github.com/Microsoft/vscode-languageserver-node/issues/105
+export function code2ProtocolConverter(value: vscode.Uri) {
```

Comment:
And we also need to account for this bug which won't be fixed since it's becomes a feature rather than a bug. Too many things depend on this behavior from VS Code.